components/board/client.go

```diff
@@ -169,7 +153,22 @@ func (ac *analogClient) Read(ctx context.Context, extra map[string]interface{})
 }
 
 func (ac *analogClient) Write(ctx context.Context, value int, extra map[string]interface{}) error {
-	return errors.New("unimplemented")
+	ext, err := protoutils.StructToStructPb(extra)
+	if err != nil {
+		return err
+	}
+	// the api method is named ReadAnalogReader, it is named differenlty than
```

Comment:
comment doesnt seem relevant?

Reply:
Whoops! Serves me right for copying and pasting. :confounded: Thanks!